WORKSPACE

```diff
@@ -15,3 +15,29 @@ envoy_dependencies()
 load("//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
```

Comment:
roll them into `envoy_dependency_imports`.

Reply:
@lizan I think I've done what you were suggesting. Local test, api build and format check look OK. Let me know if you were thinking of something different.